docs/discussions/301.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE =HTML>
+<div class=wrapper>
+<head>
+        <a href="/" style="position: fixed; top: 10px; left: 10px; display: none;" id="backLink">←Homepage</a>
+</head>
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-009HP51NPG">
+         window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'G-009HP51NPG');
+        window.addEventListener('scroll', function() {
+                var backLink = document.getElementById('backLink');
+                if (window.scrollY === 0) {
+                        backLink.style.display = 'none';
+                } else {
+                        backLink.style.display = 'block';
+                }
+        });
+</script>
+<link rel="stylesheet" type="text/css" href="/style.css">
+<script src="/star.js"></script>
+
+
+
+<h1 id="ex">Getting Started!</h1>
+<a href="/">←Back</a>
+<a href="#" font-style:italic;" onclick="copyToClipboard(window.location.href)">Copy Link</a><br>
+
+<br>
+<h1 id="intro">Intro</h1>
+If you are looking at this, you mayhaps be interesting in getting around network blocks on your device?
+
+If you are simply looking for a Prox(y) setup, this isn't for that;
+<ul>
+    <li>You can either use <a href='https://go.sillylittle.tech'>go.sillylittle.tech</a></li>
+    <li>or</li>
+    <li>append your destination to <a href="https://prox.sillylittle.tech">prox.sillylittle.tech</a> Read about these further below.</li>
```

Comment:
Perhaps link via anchor to the "below" here
```suggestion
    <li>append your destination to <a href="https://prox.sillylittle.tech">prox.sillylittle.tech</a> Read about these further <a href="#other-tools>below</a>.</li>
```
Make sure the heading is properly marked in that case then